content/doc/developer/tutorial/prepare.adoc

```diff
@@ -40,9 +40,11 @@ On macOS, the link:https://brew.sh/[Homebrew] package manager offers Maven packa
 Make sure a recent version of Maven 3, ideally 3.8.3 or newer, is provided if you decide to go this route.
 // TIMEBASED
 
-Extract Maven, and take note of its location. Then, add the the full path of the `bin/` subdirectory extracted (for example, `~/Applications/apache-maven/bin/mvn` or `C:\Program files\Maven\bin\mvn`) to the `PATH` variable in your OS. (This will let you invoke Maven using `mvn`).
+Next you will need to extract Maven, and take note of its location. When you extract the Maven files, make sure you extract them directly into the target repository (e.g. extract straight into C:\Program Files (x86)\Maven ) - do not extract the files to a different location then copy the files.
```

Comment:
The word "repository" is almost always used in this site to mean a source control repository or an artifact repository.  I think the word "directory" is a better choice in this case.

I don't understand the motivation for the advice to not copy the files into the location.  Can you describe the issue you encountered so that I better understand why you felt you needed to advise the user to not copy the files?

```suggestion
Next you will need to extract Maven, and take note of its location. When you extract the Maven files, make sure you extract them directly into the target directory (e.g. extract straight into C:\Program Files (x86)\Maven ) - do not extract the files to a different location then copy the files.
```

Reply:
I am on Windows, the issue I had is that when I copied the files and sorted the PATH out, mvn -v would not work. After some searching online I came across a few bits of advice saying to extract directly to the target folder instead of copying the files and it worked.